Consolidated tokenization logic and added documentation details (#64)
cisco/mindmeld: source/versions/changes.rst

MindMeld 4.2
-------------

> MindMeld 4.2 packages several new features to allow developers to build non-english NLP applications, do unstructured QA searches,

Reviewer: ```suggestion
MindMeld 4.2 packages several new features to make it easier for developers to build NLP applications for non-English languages, do unstructured QA searches,
```

> and convert RASA and DialogFlow conversation applications to MindMeld applications for ease of on-boarding onto the platform.

Reviewer: ```suggestion
and convert Rasa and Dialogflow conversation applications to MindMeld applications for ease of on-boarding onto the platform.
```

**4. New Human Resources Blueprint**

MindMeld 4.2 provide an enterprise Human Resources bot blueprint to complement the existing consumer blueprints we currently support. Refer to

> :ref:`hr assistant <hr_assistant>` blueprint for more details.

Reviewer: ```suggestion
:ref:`HR assistant <hr_assistant>` blueprint for more details.
```

**6. Locale and Language codes**

> MindMeld 4.2 now supports non-english system entity classification and resolution. Please see :ref:`Specify language and locale codes <specify_language>` for more details.

Reviewer: ```suggestion
MindMeld 4.2 now supports system entity classification and resolution in non-English languages. Please see :ref:`Specify language and locale codes <specify_language>` for more details.
```

**7. Stemming**

> MindMeld 4.2 allows developers to specify their own custom stemmers, which are useful for non-english languages. Please see :ref:`Stemming <stemming>` for more details.

Reviewer: ```suggestion
MindMeld 4.2 allows developers to specify their own custom stemmers, which are useful for many languages. Please see :ref:`Stemming <stemming>` for more details.
```
